src/store/file.js

```diff
@@ -1,14 +1,18 @@
+//Dependencies.
 import { CoreStore } from 'focus-core/store';
+const definition = {
+    files: 'files'
+};
 
 /**
-* Store dealing with files.
-* @type {focus}
+* Class standing for the account relative data store.
 */
-const FileStore = new CoreStore({
-    files: 'files'
-});
-
-FileStore.name = 'FileStore';
```

Comment:
By doing so you lost the name. Isn’t it gonna cause issue with the devtools ?

Reply:
nah, we don't care.
No one is using this, only for legacy.

In version 1.0.0, it should not have any link with focus core